hotham/src/resources/render_context.rs

```diff
@@ -229,26 +229,46 @@ impl RenderContext {
         let near = 0.05;
         let far = 100.0;
 
-        let fov_left = self.views[0].fov;
-        let fov_right = self.views[1].fov;
-        let fov = xr::Fovf {
-            angle_left: fov_left.angle_left,
-            angle_right: fov_right.angle_right,
-            angle_up: fov_left.angle_up,
-            angle_down: fov_left.angle_down,
-        };
+        // Compute left_frustum_from_world
+        let left_frustum_from_left_camera = get_projection(self.views[0].fov, near, far);
+        let left_camera_from_world = self.cameras[0]
+            .position
+            .to_homogeneous()
+            .try_inverse()
+            .unwrap();
+        let left_frustum_from_world = left_frustum_from_left_camera * left_camera_from_world;
 
-        // A virtual camera between the player's eyes
-        let frustum_projection = get_projection(fov, near, far);
-        let frustum_view = self.cameras[0]
+        // Compute right_frustum_from_world
+        let right_frustum_from_right_camera = get_projection(self.views[1].fov, near, far);
+        let right_camera_from_world = self.cameras[1]
             .position
-            .lerp_slerp(&self.cameras[1].position, 0.5)
             .to_homogeneous()
             .try_inverse()
             .unwrap();
+        let right_frustum_from_world = right_frustum_from_right_camera * right_camera_from_world;
 
+        // Normals of the clipping planes are pointing towards the inside of the frustum.
+        // We are only using four planes per camera. The near and far planes are not used.
+        // This link points to a paper describing the math behind these expressions:
+        // https://www.gamedevs.org/uploads/fast-extraction-viewing-frustum-planes-from-world-view-projection-matrix.pdf
+        let normalize_plane = |p: RowVector4<_>| p / p.columns(0, 3).norm();
         let cull_data = CullData {
-            frustum: frustum_projection * frustum_view,
+            clip_planes1: Matrix4::<_>::from_rows(&[
```

Comment:
I'd prefer these be called something like `left_clip_plane` and `right_clip_plane`, just to be consistent with the other variables, both here and in the compute shader